build/win32/cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "inno_updater"
+version = "0.9.0"
+authors = ["Microsoft <monacotools@microsoft.com>"]
```

Comment:
We shouldn't have their e-mail in here. I'd just remove it, as far as I know we don't have a good one to put in its place and it's easy enough for people to find ways to contact us (Twitter, Github, MS forums)